autogen.sh

```diff
@@ -5,8 +5,18 @@ autoreconf -v -f --install "$(dirname "$0")" || exit 1
 CFLAGS="-O2"
 CFLAGS="$CFLAGS -Wall"
 CFLAGS="$CFLAGS -Wextra"
+#CFLAGS="$CFLAGS -Werror=declaration-missing-parameter-type"
+#CFLAGS="$CFLAGS -Werror=deprecated-non-prototype"
 CFLAGS="$CFLAGS -Werror=implicit-function-declaration"
+CFLAGS="$CFLAGS -Werror=implicit-int"
+#CFLAGS="$CFLAGS -Werror=incompatible-function-pointer-types"
+CFLAGS="$CFLAGS -Werror=incompatible-pointer-types"
+CFLAGS="$CFLAGS -Werror=int-conversion"
+#CFLAGS="$CFLAGS -Werror=return-mismatch"
+#CFLAGS="$CFLAGS -Werror=strict-prototypes"
```

Comment:
Can you add a comment at the end of the line explaining why these are commented?

Reply:
It's commented in the commit message already.  Do you want it there too?  Or maybe I should just remove those lines?

Reply:
Thanks!